eclair-core/src/test/scala/fr/acinq/eclair/payment/relay/NodeRelayerSpec.scala

```diff
@@ -295,10 +295,11 @@ class NodeRelayerSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("appl
     incoming.foreach(p => nodeRelayer ! NodeRelay.Relay(p))
 
     val payFSM = mockPayFSM.expectMessageType[akka.actor.ActorRef]
+    router.expectMessageType[RouteRequest]
     payFSM ! Status.Failure(BalanceTooLow)
 
     incoming.foreach { p =>
-      val fwd = register.expectMessageType[Register.Forward[CMD_FAIL_HTLC]]
+      val fwd = register.expectMessageType[Register.Forward[CMD_FAIL_HTLC]] // TODO: timing out?
```

Comment:
What is this `TODO` for?

Reply:
Woops, that was my comment on where the test was failing :smile: